cli/flox-rust-sdk/src/models/manifest.rs

```diff
@@ -329,6 +336,12 @@ pub fn insert_packages(
                 descriptor_table.insert("version", Value::String(Formatted::new(version.clone())));
             }
             if let Some(ref input) = pkg.input {
+                if let Some(1) = manifest_version {
```

Comment:
```suggestion
                // TODO: catalog: drop input from `PackageToInstall`
                if let Some(1) = manifest_version {
```

Reply:
formulated it as  "drop input from `PackageToInstall` when removing support for v0 manifests" so that we dont drop it while we still somewhat support it in some cases